src/main/java/core/basesyntax/figures/Square.java

```diff
@@ -0,0 +1,39 @@
+package core.basesyntax.figures;
+
+import java.util.Locale;
+
+public class Square extends Figure {
+    private double side;
+    private double area;
+
+    public Square(String color, double side) {
```

Comment:
Creating a square, don't pass expressions in the constructor. Assign the value to a variable first for better readability.